tests/test_flip.py

```diff
@@ -674,60 +666,9 @@ def test_should_replace_pending_transactions_if_model_lowers_bid_and_increases_g
         time_travel_by(self.web3, flipper.ttl() + 1)
         assert flipper.deal(kick).transact()
 
-    def test_should_not_tend_on_rounding_errors_with_small_amounts(self, kick_small_lot):
```

Comment:
These two tests are replaced with `test_should_not_bite_dusty_urns` in `test_bite.py`.